code/modules/defenses/sentry_computer.dm

```diff
@@ -0,0 +1,487 @@
+#define DEFAULT_CAMERA_SIZE 15
+
+/**
+ * Sentry gun computer which links to defensive structures.
+ */
+/obj/item/device/sentry_computer
+	name = "\improper Sentry Gun Network Laptop"
+	desc = "A laptop loaded with sentry control software."
+	icon = 'icons/obj/structures/props/sentrycomp.dmi'
+	icon_state = "sentrycomp_cl"
+	w_class = SIZE_SMALL
+
+	var/setup = FALSE // if the laptop has been placed on a table
+	var/open = FALSE // if the laptop has been opened (the model not tgui)
+	var/on = FALSE // if the laptop is turned on and powered
+	var/list/paired_sentry = list() // list of paired defences
+	var/obj/structure/machinery/defenses/sentry/current // defensive structure which has the camera active
+	var/cell_type = /obj/item/cell/high
+	var/obj/item/cell/cell // battery in the laptop
+	var/power_consumption = 1 // multiplier for how much power to drain per linked device
+	var/list/registered_tools = list()
+	var/silent = FALSE // if the laptop should annouce events on radio, for live server testing
```

Comment:
should this still be false?

Reply:
Yes, the idea is you can set a flag on the laptop so that it does not do updates on the radio. This way if you are testing stuff - it does not flood the radio.